task2.py

```diff
@@ -0,0 +1,15 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+# Пример:
+# - [2, 3, 4, 5, 6] => [12, 15, 16];
+# - [2, 3, 5, 6] => [12, 15]
+
+
+list = [2, 3, 4, 5, 6, 1, 1]
+list2 = []
+
+for i in range(int(len(list)//2+(len(list) % 2))):
+
+    list2.append(list[i]*list[len(list)-i-1])
```

Comment:
```suggestion
for i in range((len(list) + 1) // 2):
    list2.append(list[i]*list[len(list) - i - 1])
```
Так намного проще, правда ведь? Кстати, можно и так:
```suggestion
for i in range((len(list) + 1) // 2):
    list2.append(list[i]*list[-i - 1])
```